compiler/luci/lang/include/luci/IR/Nodes/CircleArgMax.h

```diff
@@ -20,7 +20,7 @@
 #include "luci/IR/CircleNodeDecl.h"
 #include "luci/IR/CircleOpcode.h"
 
-#include "luci/IR/LuciNodeMixins.h"
+#include "luci/IR/CircleNodeMixins.h"
```

Comment:
I think `CircleArgMax` do not use neither `LuciNodeMixins` nor `CircleNodeMixins` @.@
How about just remove this header at another PR together with other same cases?

Reply:
Most Nodes derive from `FixedArityNode` which is in `CircleNodeMixins.h` as it's also a mixin.

Reply:
Oh, I didn't catch it, thanks :D